readme.md

```diff
@@ -45,15 +45,14 @@ _[Information on installing Mapbox GL for iOS normally](https://github.com/mapbo
 
 You can change the `styleURL` to any valid GL stylesheet, here are a few:
 
-* `https://www.mapbox.com/mapbox-gl-styles/styles/basic-v7.json`
-* `https://www.mapbox.com/mapbox-gl-styles/styles/bright-v7.json`
-* `https://www.mapbox.com/mapbox-gl-styles/styles/dark-v7.json`
-* `https://www.mapbox.com/mapbox-gl-styles/styles/emerald-v7.json`
-* `https://www.mapbox.com/mapbox-gl-styles/styles/light-v7.json`
-* `https://www.mapbox.com/mapbox-gl-styles/styles/mapbox-streets-v7.json`
-* `https://www.mapbox.com/mapbox-gl-styles/styles/outdoors-v7.json`
-* `https://www.mapbox.com/mapbox-gl-styles/styles/pencil-v7.json`
-* `https://www.mapbox.com/mapbox-gl-styles/styles/satellite-v7.json`
+* `asset://styles/basic-v7.json`
+* `asset://styles/bright-v7.json`
+* `asset://styles/dark-v7.json`
+* `asset://styles/emerald-v7.json`
+* `asset://styles/light-v7.json`
+* `asset://styles/mapbox-streets-v7.json`
+* `asset://styles/outdoors-v7.json`
+* `asset://styles/satellite-v7.json`
```

Comment:
In mapbox/mapbox-gl-native, we’re recommending that Cocoa developers use `-[MGLMapView bundledStyleURLs]` to discover which styles are bundled with the framework. That’s probably going to be more reliable than this list.